src/pybind/mgr/dashboard/frontend/src/app/ceph/block/iscsi-setting/iscsi-setting.component.html

```diff
@@ -1,6 +1,6 @@
 <div class="form-group"
      [formGroup]="settingsForm">
-  <label class="col-form-label"
+  <label class="cd-col-form-label"
```

Comment:
We are using vertical form in this page, by changing the class it breaks and needs fixing.
I would suggest to keep the old format:
```suggestion
  <label class="col-form-label"
```